app/models/git_hub_api/issue.rb

```diff
@@ -44,12 +46,14 @@ def add_labels(labels_input)
       GitHubApi.execute(@client, :add_labels_to_an_issue, @repo_name, @number, labels)
 
       labels.each do |l|
+        wipify_title if l == "wip"
         applied_labels[l] = Label.new(@repo, l, self)
       end
     end
 
     def remove_label(label_name)
```

Comment:
Funny that this method is singular but the other is plural (Not your issue, just jumped out at me)